frontend/src/metabase/query_builder/actions/core/core.ts

```diff
@@ -65,21 +74,26 @@ export const reloadCard = createThunkAction(RELOAD_CARD, () => {
 
     dispatch(resetQB());
 
+    if (!outdatedQuestion) {
+      return;
+    }
+
     const action = await dispatch(
       Questions.actions.fetch({ id: outdatedQuestion.id() }, { reload: true }),
     );
     const card = Questions.HACK_getObjectFromAction(action);
 
-    // We need to manually massage the paramters into the parameterValues shape,
+    // We need to manually massage the parameters into the parameterValues shape,
     // to be able to pass them to new Question.
     // We could use _parameterValues here but prefer not to use internal fields.
-    const parameterValues = outdatedQuestion.parameters().reduce(
-      (acc, next) => ({
-        ...acc,
-        [next.id]: next.value,
-      }),
-      {},
-    );
+    const parameterValues: Record<ParameterId, Parameter["value"]> =
```

Comment:
same here - I wonder if we can get `ParameterValues` to work with a few changes in that type